backend/lib/edgehog/repo.ex

```diff
@@ -35,6 +35,24 @@ defmodule Edgehog.Repo do
     Process.get(@tenant_key)
   end
 
+  def fetch(queryable, id, opts \\ []) do
+    empty = Keyword.get(opts, :error, :not_found)
```

Comment:
Should we also remove `:error` tuple from `opts` before passing them to `get`/`get_by`?
```suggestion
    {empty, opts} = Keyword.pop_first(opts, :error, :not_found)
```

Reply:
done